src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -44,7 +44,11 @@ public String getDisplayName() {
     public QAReport executeQACheck() throws Exception {
         QAReport report = new QAReport();
 
-        skipList = new SkipList(this.getDisplayName());
+        try {
+            skipList = new SkipList(this.getDisplayName());
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Just wondering, are exceptions printed in the project outside this PR?  Maybe logging would be a better approach so the contents can be directed and captured as required.

Reply:
I'm not sure what you mean - the stacktrace will point to the exact location of the issue, and I've added custom messages for `FileNotFound` and `IOException` exceptions that can occur. Could you describe what you mean in more detail, and what is missing from the current approach?

Reply:
I'm just wondering if this will only be printed to the screen and not captured in a file for later review?  In other projects, we've tended to use the `log4j2` and `logger` objects.

Reply:
Got it, OK I'll look into implementing that

Reply:
Thanks!  It's not a huge deal and shouldn't stop merging if you need to, though.  Just a thought and depends on how the rest of the project does things, too.  If code written by others prints the stacktrace, it might be worth discussing moving to log files on a developer call sometime.

Reply:
Done

Reply:
Looks good!  Just wanted to get your thoughts on if the exceptions should be logged using `logger.error()` rather than `logger.info()`?

Reply:
Oh right, yea thats my bad

Reply:
OK thats been done